cornflow-server/cornflow/tests/unit/test_main_alarms.py

```diff
@@ -0,0 +1,69 @@
+"""
+
+"""
+# TODO: Remove this
+import sys, os
+prev_dir = os.path.join(os.path.dirname(__file__), "..", "..", "..")
+print(prev_dir)
+sys.path.insert(1, prev_dir)
```

Comment:
Why do you need this?

Reply:
Because this way I can run the tests directly with Pycharm

Reply:
That is weird, you should be able to run the tests without the need for this.